jbehave-weld/src/test/java/org/jbehave/core/configuration/weld/ConfigurationProducer.java

```diff
@@ -63,19 +62,5 @@ public MyDateConverter() {
             super(new SimpleDateFormat("yyyy-MM-dd"));
         }
     }
-    
-    public static class CustomObject {
```

Comment:
what is the reason to remove this class?

Reply:
just checking :)